internal/host/host.go

```diff
@@ -272,7 +277,23 @@ func (m *Manager) HandlePrepareInstallationFailure(ctx context.Context, h *model
 	return err
 }
 
+func (m *Manager) RefreshInventory(ctx context.Context, cluster *common.Cluster, h *models.Host, db *gorm.DB) error {
+	return m.updateInventory(ctx, cluster, h, h.Inventory, db)
+}
+
 func (m *Manager) UpdateInventory(ctx context.Context, h *models.Host, inventoryStr string) error {
+	log := logutil.FromContext(ctx, m.log)
+	cluster, err := common.GetClusterFromDB(m.db, h.ClusterID, true)
```

Comment:
Please use https://github.com/openshift/assisted-service/blob/master/internal/common/db.go#L68-L73 instead of bool